File: Tienda_Pyton/Compras.py
class Compras:
    def __init__(self, cliente, producto, fecha, cantidad):
        self.cliente = cliente
        self.producto = producto
        self.fecha = fecha
        self.cantidad = cantidad

    def obtener_informacion_compra(self):
        return f"El cliente: {self.cliente.get_nombre()} compro: {self.cantidad} {self.producto.get_nombre()} el dia: {self.fecha}"

File: Tienda_JAVA/src/javaapplication45/Compras.java

package javaapplication45;

public class Compras {
    private Cliente cliente;
    private Producto producto;
    private String fecha;
    private int cantidad;

    public Compras(Cliente cliente, Producto producto, String fecha, int cantidad) {
        this.cliente = cliente;
        this.producto = producto;
        this.fecha=fecha;
        this.cantidad= cantidad;
    }
    
    public String ObtenerInformacionCompra(){
        return String.format("El cliente: %s compro: %d %s el dia: %s", cliente.getNombre(), cantidad, producto.getNombre(), fecha);
    }
}
